package fw;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.ImageView;
import androidx.appcompat.widget.AppCompatImageView;
import androidx.constraintlayout.widget.ConstraintLayout;
import com.journeyapps.barcodescanner.DecoratedBarcodeView;
import com.transsion.usercenter.R$id;
import com.transsion.usercenter.R$layout;

/* loaded from: classes7.dex */
public final class m implements g4.a {

    /* renamed from: a, reason: collision with root package name */
    private final ConstraintLayout f62898a;

    /* renamed from: b, reason: collision with root package name */
    public final AppCompatImageView f62899b;

    /* renamed from: c, reason: collision with root package name */
    public final ConstraintLayout f62900c;

    /* renamed from: d, reason: collision with root package name */
    public final ImageView f62901d;

    /* renamed from: e, reason: collision with root package name */
    public final DecoratedBarcodeView f62902e;

    private m(ConstraintLayout constraintLayout, AppCompatImageView appCompatImageView, ConstraintLayout constraintLayout2, ImageView imageView, DecoratedBarcodeView decoratedBarcodeView) {
        this.f62898a = constraintLayout;
        this.f62899b = appCompatImageView;
        this.f62900c = constraintLayout2;
        this.f62901d = imageView;
        this.f62902e = decoratedBarcodeView;
    }

    public static m a(View view) {
        int i11 = R$id.ivClose;
        AppCompatImageView appCompatImageView = (AppCompatImageView) g4.b.a(view, i11);
        if (appCompatImageView != null) {
            ConstraintLayout constraintLayout = (ConstraintLayout) view;
            i11 = R$id.v_status_space;
            ImageView imageView = (ImageView) g4.b.a(view, i11);
            if (imageView != null) {
                i11 = R$id.zxing_barcode_scanner;
                DecoratedBarcodeView decoratedBarcodeView = (DecoratedBarcodeView) g4.b.a(view, i11);
                if (decoratedBarcodeView != null) {
                    return new m(constraintLayout, appCompatImageView, constraintLayout, imageView, decoratedBarcodeView);
                }
            }
        }
        throw new NullPointerException("Missing required view with ID: ".concat(view.getResources().getResourceName(i11)));
    }

    public static m c(LayoutInflater layoutInflater) {
        return d(layoutInflater, null, false);
    }

    public static m d(LayoutInflater layoutInflater, ViewGroup viewGroup, boolean z10) {
        View inflate = layoutInflater.inflate(R$layout.activity_profile_qr_code, viewGroup, false);
        if (z10) {
            viewGroup.addView(inflate);
        }
        return a(inflate);
    }

    @Override // g4.a
    /* renamed from: b, reason: merged with bridge method [inline-methods] */
    public ConstraintLayout getRoot() {
        return this.f62898a;
    }
}
